CHANGELOG.md

```diff
@@ -1,5 +1,42 @@
 # Changelog
 
+## [1.49](https://github.com/brave/brave-ios/milestone/106)
```

Comment:
We'll need to point to a tag/entry within https://github.com/brave/brave-ios/releases rather than the milestone. For example:

* https://github.com/brave/brave-ios/releases/tag/v1.34.1
* https://github.com/brave/brave-ios/releases/tag/v1.34

It's been a while so going to ping @iccub and will include you as well.